ci/hack/rosa_cluster_provision.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+#quit if exit status of any cmd is a non-zero value
+set -euo pipefail
+
+# Uncomment the below line to enable debugging
+# set -x
+
+tmpdir=$(mktemp -d)
+cd "$tmpdir"
+
+# This repo has the rosa provision and destroy cluster scripts
+git clone git@github.com:stolostron/bootstrap-ks.git
+cd bootstrap-ks/rosa
+
+# Set the required environment variable before provisioning
+export ROSA_TOKEN=${ROSA_TOKEN:-"<your-rosa-token>"}
```

Comment:
Question: Do we want to read those values from a yaml and upload that yaml to Bitwarden? Or are those values something we do not even want to share at the team level?

Reply:
Yes we are not sharing those values at team level